test/test_sessions.rb

```diff
@@ -124,4 +124,21 @@ def test_session_with_no_users
   ensure
     KBSecret::Config.deconfigure_session(label)
   end
+
+  def test_import_record
```

Comment:
Could you add a test for the error case as well (either in another method or in another `temp_session` block in this method)?